src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -873,6 +873,8 @@ internal ImmutableArray<RefKind> ParameterRefKinds
             get;
         }
 
+        internal virtual ImmutableArray<INamedTypeSymbol> CallingConventionTypes => ImmutableArray<INamedTypeSymbol>.Empty;
```

Comment:
>INamedTypeSymbol [](start = 40, length = 16)

It looks like the API layering is violated here. Internal symbols should not operate in INamedTypeSymbol terms

Reply:
Internally we can keep operating in terms of CustomModifiers and go to types in the public API translation layer.

---
In reply to: [463851591](https://github.com/dotnet/roslyn/pull/46467#discussion_r463851591) [](ancestors = 463851591)